other/web-js/client/src/data-api.js

```diff
@@ -0,0 +1,219 @@
+import { ClientApi } from "./client-api";
+
+/**
+ * Connects to a MongoDB Atlas App Services App's Data API over HTTPS
+ */
+export class DataApi {
+  static constructBaseUrl(appId, location) {
+    const { deployment_model, cloud, region } = location;
+    if (deployment_model === "LOCAL") {
+      return `https://${region}.${cloud}.data.mongodb-api.com/app/${appId}/endpoint/data/v1/`;
+    } else {
+      return `https://data.mongodb-api.com/app/${appId}/endpoint/data/v1/`;
+    }
+  }
```

Comment:
this seems somewhat duplicative of the logic in https://github.com/mongodb-university/realm-template-apps/pull/142/files#diff-3e277fd749d5f7165b2ef838367404e1143d07bc50d05ef64a0b63f2676f33eeR7-R18.

is there a way to consolidate these?